src/pages/JazzBar/Jazz/Jazz.tsx

```diff
@@ -237,13 +238,34 @@ const Jazz: React.FunctionComponent<PropsType> = ({
         !usersSeated.includes(user)
     );
 
+  const createReaction = (type: ReactionType, user: FUser, text?: string) => ({
+    reaction: type,
+    created_at: new Date().getTime(),
+    created_by: user.uid,
+    text: text || "",
```

Comment:
Can you make this undefined? And maybe make the Type more precise (only `messageToBand` can have a text)